Cargo.toml

```diff
@@ -62,6 +62,7 @@ futures-channel = { version = "0.3.29" }
 futures-timer = { version = "3.0.2" }
 futures-util = { version = "0.3.29" }
 lru = { version = "0.12.1" }
+webbrowser = { version = "0.8.12", features = ["hardened", "disable-wsl"] }
```

Comment:
as of [their docs page](https://docs.rs/webbrowser/latest/webbrowser/):
- `hardened` - this disables handling of non-http(s) urls (e.g. file:///) as a hard security precaution
- `disable-wsl` - this disables WSL file implementation (http still works)

I've turned on these features as we don't want/need to open files on browser.